pylint/checkers/stdlib.py

```diff
@@ -444,7 +444,7 @@ class StdlibChecker(DeprecatedMixin, BaseChecker):
         ),
     }
 
-    def __init__(self, linter=None):
+    def __init__(self, linter=None):  # pylint: disable=super-init-not-called
```

Comment:
```suggestion
    def __init__(self, linter: PyLinter = None):  # pylint: disable=super-init-not-called
```
Maybe we should add a link to the issue with the `pylint: disable` statement.
Same for the other time, I think in `exceptions.py`.